app/controllers/api/v1/credit_notes_controller.rb

```diff
@@ -77,6 +77,25 @@ def download
         head(:ok)
       end
 
+      def void
+        credit_note = current_organization.credit_notes.find_by(id: params[:id])
```

Comment:
Does it make sense to have an external id here? 🤔

Reply:
Well like invoice it will be mostly created automatically, and it does not belongs to a resource from the customer system, so I don't really think it make sense